### PR TITLE
support image tag for svg

### PR DESCRIPTION
**Pull request by dujuanxian**
basically a repeat for a merge-failed pull request https://github.com/facebook/react/pull/2263

### CODE REVIEW

File: docs/js/react.js [hunk 1]
```diff
@@ -7364,6 +7364,7 @@ var ReactDOM = mapObject({
   ellipse: 'ellipse',
   g: 'g',
   line: 'line',
+  image: 'image',
```
Comment (zpao) [inline docs/js/react.js]: Can you revert the changes to this file, we only update this on official releases.

**Comment by framp** [comment]
The only hack needed is `getMarkupWrap` but adding `image` and maintaining a white-list of svg tags is not a future proof solution

See PR #3808 and related issues
Discussion here: #1657

File: docs/js/react.js (hunk 1 above)
Comment (dujuanxian) [inline docs/js/react.js]: Hi @zpao I have reverted it.

Comment (zpao) [inline docs/js/react.js]: It looks like you reverted the wrong thing. The changes in `src/` are appropriate. The change in `docs/` (the file we're commenting in right now) is not.

**Comment by kulicuu** [comment]
+1

**Comment by zpao** [comment]
Closed with #4157
